internal/test-charm-repo/quantal/multi-series/revision

```diff
@@ -0,0 +1 @@
+7
```

Comment:
This whole dir was copied from an existing charm and this file was not edited at all.